CHANGELOG.rst

```diff
@@ -18,3 +18,7 @@ Bug Fixes
 - A ``spinw`` object can now be correctly created from a structure figure
 - ``.cif`` files with a mixture of tabs and spaces or containing a ``?``
   in the comments can now be read correctly
+- Rotation matrix ``rotC``  in ``addtwin`` is now required to be a valid rotation or reflection matrix.
+- Spin of atom in ``addatom`` must have ``S>=0``.
+- Anisotropic g-tensor in ``addg`` must be physically valid - i.e. :math:`g^\dagger.g` must be a symmetric positive definite matrix.
+- Fix bug in addcoupling that did not allow user to supply 'atom' with numeric array of atom indices (previously only worked for string or cell of strings corresponding to atom labels).
```

Comment:
It's not so strict, but try and keep line length to 79 characters or less (https://documentation-style-guide-sphinx.readthedocs.io/en/latest/style-guide.html#line-length this is for Sphinx, but it can't hurt)

Reply:
Thanks didn't know about this, will split it up and avoid this in future!